landoapi/repos.py

```diff
@@ -215,6 +219,17 @@ def __post_init__(self):
             push_path="ssh://autolandhg.devsvcdev.mozaws.net//local_repos/m-c",
             pull_path="https://hg.mozilla.org/integration/autoland",
         ),
+        # A repo to test autoformatting
+        "formatting-repo": Repo(
+            tree="autoformatting",
+            url="https://autolandhg.devsvcdev.mozaws.net/local/autoformatting/",
+            access_group=SCM_VERSIONCONTROL,
+            pull_path="https://autolandhg.devsvcdev.mozaws.net/local/autoformatting/",
+            formatting={
+                "black:command": "black -t py36 -- -",
+                "black:pattern": "set:**.py",
```

Comment:
I think these would be cleaner placed as normal config overrides, as
```suggestion
            formatting={
                "fix.black:command": "black -t py36 -- -",
                "fix.black:pattern": "set:**.py",
```

We would also include:
```
config_overrides = {
    "extensions.fix": "",
    "fix.failure": "abort",
},
```

And merge them within `HgRepo` into the same eventual config dict.

Reply:
> I think it would be a bit cleaner to merge this config within HgRepo.__init__ rather than doing it here.

Thought of that but since we don't have a `repo` object when calling the constructor, we can't set up the config override there. :/